FuelSDK-Test/RefreshTokenTest.cs

```diff
@@ -0,0 +1,109 @@
+﻿using NUnit.Framework;
+using System;
+using FuelSDK;
+using Newtonsoft.Json.Linq;
+using System.Reflection;
+using System.Collections.Specialized;
+
+namespace FuelSDK.Test
+{
+    [TestFixture()]
+    public class RefreshTokenTest
+    {
+        ETClient client;
+        FuelSDKConfigurationSection config;
+ 
+        [OneTimeSetUp]
+        public void Setup()
+        {
+            client = new ETClient();
+            config = new FuelSDKConfigurationSection();
+            config.AuthorizationCode = "Auth_Code_For_OAuth2_WebApp";
+            config.RedirectURI = "www.google.com";
+            config.ClientId = "OAUTH2_CLIENTID";
+            config.ClientSecret = "OAUTH2_CLIENT_SECRET";
+            
+        }
+
+        [Test()]
+        public void AuthTokenShouldDifferIfRefreshTokenIsEnforced()
```

Comment:
- If I would see just the name of the test method, I would expect the method to have just the first Assert.
Since we are testing the fact that the access token received in the response of the second auth call, made by passing the Refresh Token is different than the one obtained in the first call, maybe we should keep just the first Assert or rename the test method.
-  Do you think it would be useful to add a comment in this test suite that would highlight the fact that these tests are relevant only for configurations using OAuth2 and that the first test is relevant just for Web/Public API integrations? I think this comment would spare the person running these tests from trying to understand how these tests are different from the others (since these tests are quite particular).